rust/projects/project-2/project.md

```diff
@@ -188,6 +192,22 @@ this should be straightforward. Add panics anywhere necessary to get the test to
 compile (`cargo test --no-run`).
 
 
+## Aside: The history of Rust error handling
+
+_Note: Error-handling practices in Rust are still evolving. This course
+currently uses the [`failure`] crate to make defining error types easier. While
+`failure` has a good design, it's use [arguably not a best practice][nbp]. It
+may not continue to be viewed favorably by the Rust zeitgeist. Future iterations
```

Comment:
Is `zeitgeist` a simple English word?

Reply:
lol no. I knew that while I was writing it... I'll figure something else out. Thanks for keeping me honest.